antarest/study/web/raw_studies_blueprint.py

```diff
@@ -513,11 +510,12 @@ def edit_study(
     def replace_study_file(
         uuid: str,
         path: str = Param("/", examples=get_path_examples()),  # type: ignore
-        file: bytes = File(...),
+        file: bytes = File(default=b""),
```

Comment:
Okay but what's the behavior you expect when someone doesn't fill this parameter ? As for empty bytes it will simply erase all content inside an existing file or create an empty one if create_missing=True

Reply:
I think we should throw a 4XX if we have a `file` argument with a `ResourceType.FOLDER`, or if we have no `file` argument with a `ResourceType.FILE`.
To do those checks we need to differentiate between "empty but present" and "absent".